test/acceptance/keywords/tables/table_should_not_contain.robot

```diff
@@ -0,0 +1,120 @@
+*** Settings ***
+Documentation     Tests table contents for should not keyowrds
+Resource          table_resource.robot
+Default Tags  table_not_contain
+
+
+*** Test Cases ***
+Should Give Error Message When Content Found In Table
+    [Documentation]    Should Give Error Message When Content Found In Table
+    [Template]    Verify Table Does Contain With CSS And XPath Locators
+    simpleTable    simpleTable_A1
+    simpleTable    simpleTable_C3
+    simpleWithNested    simpleWithNested_A1
+    simpleWithNested    nestedTable_A1
+    nestedTable    nestedTable_A1
+    nestedTable    nestedTable_C3
+    tableWithSingleHeader    tableWithSingleHeader_B1
+    tableWithSingleHeader    tableWithSingleHeader_B2
+    withHeadAndFoot    withHeadAndFoot_AH1
+    withHeadAndFoot    withHeadAndFoot_B2
+    withHeadAndFoot    withHeadAndFoot_CF1
+    mergedRows    mergedRows_A1
+    mergedCols    mergedCols_B2
+    formattedTable    formattedTable_A1
+    formattedTable    formattedTable_B1
+    formattedTable    formattedTable_C1
+    formattedTable    formattedTable_A2
+    formattedTable    formattedTable_B2
+    formattedTable    formattedTable_ÄÖÜäöüß
+    formattedTable    äöü€&äöü€&
+
+Test Table Cell Should Not Contain
+    [Documentation]    Test Table Cell Should Not Contain
+    ${table}=    set variable    simpleTable
+    ${row}=    set variable    1
+    ${col}=    set variable    1
+    ${value} =    set variable    simpleTable_A2
+    table cell should not contain    ${table}    ${row}    ${col}    ${value}
+
+Test Table Cell Give Error When Cell Not Present
+    [Documentation]    Test Table Cell Should Not Contain when Cell is not present in the table
+    ${table}=    set variable    simpleTable
+    ${row}=    set variable    1
+    ${col}=    set variable    10
+    ${value} =    set variable    simpleTable_A1
+    ${err}=    Set Variable    Cell in table '${table}' in row #${row} and column #${col} was not found
+    Run Keyword And Expect Error    ${err}    table cell should not contain    ${table}    ${row}    ${col}    ${value}
+
+Test Table Cell Give Error When It Contains
+    [Documentation]    Test Table Cell Give Error When It Contains
+    ${table}=    set variable    simpleTable
+    ${row}=    set variable    1
+    ${col}=    set variable    1
+    ${value} =    set variable    simpleTable_A1
+    ${err}=    Set Variable    Cell in table '${table}' in row #${row} and column #${col} should not have contained text '${value}'.
+    Run Keyword And Expect Error    ${err}    table cell should not contain    ${table}    ${row}    ${col}    ${value}
+
+Test Table Row Should Not Contain
```

Comment:
Value is an exact match, right? Expand test partial text too. Also add test when value is in table header and footer.